samples/kubeflow/README.md

```diff
@@ -1,6 +1,7 @@
 # Kubeflow Jupyter notebook demo
 
```

Comment:
@tomersolomon1 there are a lot of steps for each persona. Did you consider formating as steps? would reduce the need to use "then", "before" --- mostly applies to the data user part.

Reply:
Cool, added these, see last updates